HealthMonitoring.AcceptanceTests/HealthMonitoring.AcceptanceTests.csproj

```diff
@@ -10,11 +10,12 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>HealthMonitoring.AcceptanceTests</RootNamespace>
     <AssemblyName>HealthMonitoring.AcceptanceTests</AssemblyName>
-    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
I had to update test projects to 4.6, otherwise xunit 2.2.0 will put reference to NetStandard packages which would install tens of System packages in each project.